test/unit/acc/src/AccTagTest.cpp

```diff
@@ -0,0 +1,269 @@
+/* Copyright 2022 Simeon Ehrig
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/acc/TestAccs.hpp>
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+
+#include <string>
+
+
+using Dim = alpaka::DimInt<1>;
+using Idx = int;
+using TestAccs = alpaka::test::EnabledAccs<Dim, Idx>;
+
+using TagList = std::tuple<
+    alpaka::AccCpuSerialTag,
+    alpaka::AccCpuThreadsTag,
+    alpaka::AccCpuFibersTag,
+    alpaka::AccCpuTbbBlocksTag,
+    alpaka::AccCpuOmp2BlocksTag,
+    alpaka::AccCpuOmp2ThreadsTag,
+    alpaka::AccOmp5Tag,
+    alpaka::AccOaccTag,
+    alpaka::AccGpuCudaRtTag,
+    alpaka::AccGpuHipRtTag,
+    alpaka::AccCpuSyclIntelTag,
+    alpaka::AccFpgaSyclIntelTag,
+    alpaka::AccFpgaSyclXilinxTag,
+    alpaka::AccGpuSyclIntelTag>;
+
+// to many acc's triggers a compiler error in nvc++: error: excessive recursion at instantiation of class
+// looks like, that a buffer is not big enough
+#ifdef ALPAKA_ACC_ANY_BT_OACC_ENABLED
+using AccToTagMap = std::tuple<
+    std::tuple<alpaka::test::detail::AccCpuSerialIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuSerialTag>,
```

Comment:
Is there a reason why you chose `std::tuple<std::tuple<X, Y>>` over `std::tuple<std::pair<X, Y>>`?